tox.ini

```diff
@@ -45,12 +42,57 @@ commands =
     coverage run -p kolibri stop
     sh -c 'kolibri manage makemigrations --check'
     # Run the actual tests
-    py.test {posargs:--cov=kolibri --color=no}
+    py.test {posargs:--cov=kolibri --cov-report= --cov-append --color=no}
```

Comment:
1) Suppressed the annoying coverage report output in Travis @jonboiser 
2) Appends to an existing report, such that results from above coverage isn't overwritten.